spec/module.dd

```diff
@@ -219,26 +219,46 @@ $(GNAME ModuleAliasIdentifier):
 	with whatever packages they are in. They are not considered to
 	be relative to the module that imports them.)
 
-$(H3 $(LNAME2 basic_imports, Basic Imports))
+$(H3 $(LNAME2 name_lookup, Symbol Name Lookup))
 
 	$(P The simplest form of importing is to just list the
 	modules being imported:)
 
 ---------
+module myapp.main;
+
 import std.stdio; // import module stdio from package std
-import foo, bar;  // import modules foo and bar
```

Comment:
This somewhat hides the comma-separation syntax. Do we actually want to discourage that? Fine with me. :smile: